### PR TITLE
Update layout

### PR DESCRIPTION
**Pull request by batteries76**
## Overview of the Pull Request:
Just took some padding off. There will be more changes, but this is a small thing that maybe is better. 
I'll have to get clearer what else is not quite right in the layout.

## How Has This Been Tested?
- Please describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce. 
- If necessary, please also list any relevant details for your test setup/configuration

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [ ] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [ ] Is your pull request up-to-date with `main` branch?
- [ ] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [ ] Have you written instructions on how to test that your changes work as expected?
- [ ] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

**Comment by hqtan** [comment]
Hi @batteries76 ,
thanks for your PR! :bow: 

There are some Pros and Cons that I can see with the changes:

Pros:
- taking off the padding seems to have eliminate the need to scroll to see 'Next' button in many cases (for desktop views) :tada:

Cons:
- the cloud in the 'ThankYou' page gets 'chopped off' at the top (see screenshot)

image

We can address the issue with the 'ThankYou' page on a separate PR, if we're happy to remove the padding. What do you think @Amutha37 , @OlaShabalina ?

The changes in the PR doesn't address the issues we see in mobile view for the `WorkArrangement` page and the `TravelDays` page (see screenshots below). But if they can't be easily address in `components/Layout/Layout.jsx`, then let's address the layout issues in the pages.

mobile view of WorkArrangement page


mobile view of TravelDays page

**Comment by batteries76** [comment]
Yeah, I think perhaps the issues with pages should be solved within the pages. It's a bit presumptive of the layout to put that padding in (which is probably my fault in the first place).

**Comment by OlaShabalina** [comment]
@batteries76 , @hqtan 
Should we keep the original padding for mobile version? The issue we faced was for desktop. Also I don't think it should be 0, it's good to maintan a similar width for pages, just not as much as 97px. Ideally to keep around 720-750px for the content for the largest page size.

**Comment by hqtan** [comment]
> @batteries76 , @hqtan Should we keep the original padding for mobile version? The issue we faced was for desktop. Also I don't think it should be 0, it's good to maintan a similar width for pages, just not as much as 97px. Ideally to keep around 720-750px for the content for the largest page size.

Hi @OlaShabalina ,
Thx for your suggestion! :bow: 

I'm not sure I quite understand what you mean, sorry! Can you suggest the code changes you would make to implement your suggestions?

**Comment by batteries76** [comment]
Yes, good idea. 
I'll double check these things today and push a new commit.
Thanks @OlaShabalina and @hqtan

**Comment by OlaShabalina** [comment]
Hi @batteries76 , 
Thank you submitting your PR. I personally like the current **px** padding (will wait what @hqtan and @Amutha37 say). All the pages are adjusted to it already so it's better to proceed with the current state. What would be great is to remove the bottom padding so we don't see the scroll bar for the empty space.
image

**Comment by OlaShabalina** [comment]
I pulled the changes to my machine and, although the code is a bit different, how does it change the current page look/functionality? https://council-emissions-calculator-spike-flax.vercel.app/form/WorkArrangement 

One thing I noticed is - it adds more space between lines:
image

In my opinion, the current version with less padding looks better. I'll wait for response from @hqtan and @Amutha37

**Comment by batteries76** [comment]
It might be my fault, but I don't think you are looking at the right thing @OlaShabalina. I've double checked.
The easiest one to look at is [8a2085a](https://github.com/CodeforAustralia/council-emissions-calculator/pull/115/commits/8a2085a487c33f46356292505e14e4cc291ceb01), because I might have run prettier in a way that isn't how people have been using it.
There are only a couple of files changed, and the padding has been reduced in the layout and in the 'Work Arrangements' page.
I also removed the `<br>`s and put in some Flexbox. 
A couple of other minor changes.

**Comment by OlaShabalina** [comment]
Hi @batteries76 , it's possible 😄 I'll wait for @hqtan and @Amutha37 to have a look. 👍

**Comment by hqtan** [comment]
Hi @batteries76 ,

If you're fixing formatting on a bunch of files in this repo, that is unrelated to the purpose of this PR.
It makes it hard for reviewers to tease out the changes are relevant to the layout updates.

 I suggest you revert the formatting commit in this PR, and raise a separate PR for the formatting fixes.

**Comment by batteries76** [comment]
@hqtan I'm not trying to fix formatting issues. I just thought we were running prettier these days. If not fine, I'll revert.

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 yarn.lock
+package-lock.json
```
Comment (hqtan) [inline .gitignore:8]: Hi @batteries76 ,

what's the reasoning behind choosing to ignore `package-lock.json` ?
Did you mean to add this file to `.prettierignore` instead?

**Review by hqtan — CHANGES_REQUESTED** [review]

**Comment by hqtan** [comment]
Hi @batteries76 ,
the cloud in the 'ThankYou' page is still getting cropped:
image

File: .gitignore (hunk 1 above)
Comment (batteries76) [reply .gitignore:8]: It's a generated file and should be ignored. It changes when you run `npm install` and so the changes are not useful to see in the diffs. Changes made to dependencies will be reflected in the `package-lock`.

**Comment by batteries76** [comment]
Have made some changes as requested, and also spot fixed a few styling things as I went through the app.
Let me know how that looks now.

**Comment by batteries76** [comment]
Ok, second try. Git is still a struggle sometimes.
Some changes as requested, some spot fixes of various styling issues, and a few copy changes.

File: components/DaysOfTheWeek/DaysOfTheWeekContainer.jsx [hunk 2]
```diff
@@ -103,16 +103,16 @@ export default function DaysOfTheWeekContainer({
       borderRadius="8px"
       borderColor={["white", "gray.200"]}
       width={["375px", "700px"]}
-      height={["516px", "320px"]}
-      mt={10}
+      // height={["516px", "320px"]}
```
Comment (hqtan) [inline components/DaysOfTheWeek/DaysOfTheWeekContainer.jsx:106]: please remove comment
```suggestion
```

File: pages/form/Thankyou.jsx [hunk 3]
```diff
@@ -32,8 +31,8 @@ export default function Thankyou() {
             textAlign={["center", "start"]}
             w="100%"
           >
-            We’re collecting responses from 26th April 2022 to better understand 
-            how City staff travel to and from work.
+            We are collecting responses from 26th April 2022 XXXXX to better understand 
+            how City (capital? XXXX) staff travel to and from work.
```
Comment (hqtan) [inline pages/form/Thankyou.jsx:35]:
```suggestion
            We are collecting responses from 26th April 2022 to better understand 
            how Council staff travel to and from work.
```

File: .gitignore (hunk 1 above)
Comment (batteries76) [reply .gitignore:8]: I didn't mean ignored generally, it's just that it's a generated file that was showing up in the git diffs. I can see that there are issues with not including it, and ways around them.

**Comment by batteries76** [comment]
> Hi @batteries76 ,
> the cloud in the 'ThankYou' page is still getting cropped:
> image

I'm not seeing this at the moment

**Comment by batteries76** [comment]
The other changes I have put through

**Review by hqtan — APPROVED** [review]

**Review by hqtan — APPROVED** [review]
Thx for your PR @batteries76 !
thumbsup

**Pull request merged**